lib/class-wp-rest-widget-updater-controller.php

```diff
@@ -76,53 +75,107 @@ public function compute_new_widget_permissions_check() {
 	}
 
 	/**
-	 * Returns the new widget instance and the form that represents it.
+	 * Checks if the widget being referenced is valid.
 	 *
 	 * @since 5.2.0
-	 * @access public
+	 * @param string  $identifier         Widget id for callback widgets or widget class name for class widgets.
+	 * @param boolean $is_callback_widget If true the widget is a back widget if false the widget is a class widget.
 	 *
-	 * @param WP_REST_Request $request Full details about the request.
-	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 * @return boolean True if widget being referenced exists and false otherwise.
 	 */
-	public function compute_new_widget( $request ) {
-		$url_params = $request->get_url_params();
-
-		$widget = $request->get_param( 'identifier' );
-
+	private function isValidWidget( $identifier, $is_callback_widget ) {
```

Comment:
Is there a reason for this not be named using `snake_case` ?

>Use lowercase letters in variable, action/filter, and function names (never camelCase). Separate words via underscores.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions

Reply:
Hi @aduth the name was updated the snake case.